docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -82,6 +82,7 @@ chmod 770 /etc/gitea
 chmod 750 /etc/gitea
 chmod 644 /etc/gitea/app.ini
 ```
+If you don't want the web installer to be able to write the config file at all, it is also possible to make the config file read-only for the gitea user (owner/group `root:root`, mode `0660`), and set `INSTALL_LOCK = true`. In that case all database configuration details must be set beforehand in the config file, as well as the `SECRET_KEY` and `INTERNAL_TOKEN` values. See the [command line documentation](https://docs.gitea.io/en-us/command-line/) for information on using `gitea generate secret INTERNAL_TOKEN`.
```

Comment:
Can you use the Hugo `relref` function here?
https://gohugo.io/content-management/cross-references/

`[command line documentation]({{< relref "doc/usage/command-line.en-us.md" >}})`